src/js/InstitutionContainer.jsx

```diff
@@ -7,6 +7,7 @@ var InstitutionStatus = require('./InstitutionStatus.jsx');
 var InstitutionContainer = React.createClass({
 
   getInitialState: function(){
+    console.log('getting initial');
```

Comment:
Can you remove the logging? If you need it for the next PRs that's ok, this can just be a reminder then.